sqlobject/src/main/java/org/jdbi/v3/sqlobject/customizer/internal/TimestampedConfig.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.sqlobject.customizer.internal;
+
+import java.time.Clock;
+
+import org.jdbi.v3.core.config.JdbiConfig;
+
+public class TimestampedConfig implements JdbiConfig<TimestampedConfig> {
```

Comment:
It looks to me the change in the `@Timestamped` annotation should be a part of a separate PR.

Reply:
This config class is in the `internal` package, so it's not actually part of the public API.

Reply:
Although the change to make `@Timestamped` applicable to the type, that part should be separate.

Yeah, I'll revert. _sigh_